pkg/policy/opal/opal_test.go

```diff
@@ -31,3 +38,41 @@ func TestPoliciesFail(t *testing.T) {
 	assert.Equal(1, tm.Failed)
 	assert.Equal(1, tm.Passed)
 }
+
+func TestGetCustomPoliciesDir204(t *testing.T) {
+	assert := assert.New(t)
+	apiConfig := &api.Config{
+		APIServer:        "https://api.test",
+		APIPrefix:        "/api/v1",
+		Organization:     "1234",
+		LegacyAPIToken:   "token",
+		LaceworkAPIToken: "token",
+	}
+	c := api.NewClient(apiConfig)
+	o := &tools.AssessmentOpts{
+		ToolOpts: tools.ToolOpts{
+			RunOpts: tools.RunOpts{
+				PrintClientOpts: options.PrintClientOpts{
+					ClientOpts: options.ClientOpts{
+						Client: &api.Client{
```

Comment:
ClientOpts has an api.Config, so you can just configure that here.

Also given that this is testing ClientOpts.GetCustomPoliciesDir() it should be in the same package, not in opal.

Though maybe ClientOpts should be refactored a bit to make it easier to test.

Reply:
Thanks @slshen, I've tidied up this test.
However, I have it in the opal package as it requires a Tool to be specified. Doing this in the `tools` package causes an import cycle that is not allowed.  
Possibly we need to add some way of specifying a tool for testing purposes that wont cause issues?